SL/Form.pm

```diff
@@ -1443,8 +1467,8 @@ sub format_line {
 
 		s/<%(.+?)%>/$newstr/;
 
-		s/(�)/\\textsuperscript{2}/;
-		s/(�)/\\textsuperscript{3}/;
+		s/(�)/\\textsuperscript{2}/;
```

Comment:
looks like you changed the file encoding. Plz revert.

Reply:
done